pypesto/result/sample.py

```diff
@@ -48,71 +142,81 @@ def __init__(
         effective_sample_size: float = None,
         message: str = None,
     ):
-        super().__init__()
+        super().__init__(
+            trace_x,
```

Comment:
I usually prefer kwargs in long arg lists, as otherwise assuring correct order may be prone to future errors.